qi-lib/flow/core/compiler/deforest/fusion.rkt

```diff
@@ -7,7 +7,9 @@
          syntax/parse
          "syntax.rkt"
          "../../passes.rkt"
-         "../../strategy.rkt")
+         "../../strategy.rkt"
+         (for-template "../../passes.rkt"))
+
```

Comment:
RE 33ae16deaf0d1f0dfa6dc62af5094cd6fe1236fb: What commit is this fixing? It should get squashed there.

Reply:
Addressed during all those rebases.